src/components/callout/callout.tsx

```diff
@@ -18,6 +18,7 @@ import { Languages } from '@limetech/lime-elements';
  * presented. In such cases, a callout should not be used as a static and
  * constantly present element of the UI. Rather, it should be displayed when
  * something unusual or remarkable demands the user's attention.
+ *
```

Comment:
Lint error, here. `npm i` and then reload window.
<img width="626" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/03e86592-69ab-4335-bda4-5b8baff7adb7">